src/redisearch.h

```diff
@@ -16,6 +16,7 @@
 #include <time.h>
 #include "util/dllist.h"
 #include "stemmer.h"
+#include "inverted_index_rs.h"
```

Comment:
Looking at where the type was defined in C, we probably want a `type_rs.h` rather than `inverted_index_rs.h` for these specific definitions that are not used exclusively by the inverted index.